about/faq.md

```diff
@@ -0,0 +1,162 @@
+---
+title: FAQ
+layout: info
+---
+
+## {{ page.title }}
+{: .page-header .no_toc}
+
+*These FAQs primarily target LLNL developers working in the [LLNL GitHub organization](https://github.com/LLNL). Don't see your question listed below? Please contact [LLNL GitHub admins](mailto:github-admin@llnl.gov).*
+
+* Table of Contents
+{:toc}
+
+### How do I set up a GitHub account?
+
+If you’re new to GitHub and open source in general, figuring out how to get set up can be a challenge. You may want to read through the GitHub Help pages on [Setting up and managing your GitHub profile](https://help.github.com/categories/setting-up-and-managing-your-github-profile/).
+
+1. [Create an account on GitHub](https://github.com/join).
+
+    You *do not need* a separate work account and personal account. Instead, you can [link multiple email addresses to the same GitHub account](https://help.github.com/articles/adding-an-email-address-to-your-github-account/), which is almost always preferred.
+
+2. [Update your profile information](https://github.com/settings/profile).
+
+    * **Photo**: A headshot photo, or image that is uniquely you.
+    * **Name**: Your first and last name.
+    * **Bio**: Include a few words about yourself! Don't forget to mention @LLNL.
+    * **URL**: This might be your [people.llnl.gov](https://people.llnl.gov) page, or a personal website if you prefer.
+    * **Company**: Probably `Lawrence Livermore National Laboratory, @LLNL`.
+    * **Location**: Your primary location.
+
+3. Add your `@LLNL` email address (and any aliases) to your [Email Settings](https://github.com/settings/emails) page. This will link any commits done via [your Git identity](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup#Your-Identity) to your GitHub account.
+
+4. [Enable two-factor authentication (2FA)](https://github.com/settings/security).
+
+### What is two-factor authentication (2FA)?
+
+*Membership in the @LLNL GitHub organization will soon require that 2FA has been enabled on your GitHub account.* There are several options for configuring 2FA for your GitHub account:
+
+* [YubiKey](https://yubico.com) hardware security keys
+    * Note that the YubiKey deployment (called "MyPass" at LLNL) is in its pilot phase, and not all users have them yet.
+        * YubiKeys are the preferred 2FA option, due to the security of the YubiKey hardware tokens. If you have been issued your LLNL YubiKeys they are highly recommended for securing your GitHub (and other) accounts which support them.
+    * [Learn more about setting up your Yubikey with GitHub.com](https://help.github.com/articles/configuring-two-factor-authentication/#configuring-two-factor-authentication-using-fido-u2f)
```

Comment:
I would suggest changing "Yubikey" to "YubiKey" on this line to match the other instances of the word.

Functionally and visually, things look good to me. I'll leave additional content analysis to @IanLee1521

Reply:
Fixing